driver-core/src/main/com/mongodb/MongoClientSettings.java

```diff
@@ -58,7 +61,10 @@
 @Immutable
 public final class MongoClientSettings {
     private static final CodecRegistry DEFAULT_CODEC_REGISTRY =
-            fromProviders(asList(new ValueCodecProvider(),
```

Comment:
I thought this was kind of ugly, but the alternative is to create a Provider for StringCodecs, just so they can be added to the default registry

Reply:
Looking at the BsonCodecProvider and GridFSFileCodecProvider they are both very simple providers - so it may be worth adding one. Ultimately,  it doesn't really matter but having a single registry rather than two as default may be marginally beneficial.

Reply:
Done